### ISSUE TITLE
[question] Does LightGBM currently support distributed validation?

### ISSUE THREAD

**Issue by HonLZL** [microsoft/LightGBM#6956]
Hi~
In my task, a single node cannot hold the validation set. Regarding the AUC metric, does LightGBM currently support distributed validation?
Thanks!

**Comment by jameslamb** [comment]
Thanks for using LightGBM.

LightGBM does not currently support aggregated validation metrics across all workers in distributed training. However, @szjunma is experimenting with an approach for that in #7290 (in the Python package specifically, for #6963). Subscribe there to be notified how that goes.

It's looking like if it gets merged, it'd set up a good foundation for doing this generically across the CLI, R package, etc. (you did not tell us which language/interface you're using).

**Comment by honlzl** [comment]
> Thanks for using LightGBM.
> 
> LightGBM does not currently support aggregated validation metrics across all workers in distributed training. However, [@szjunma](https://github.com/szjunma) is experimenting with an approach for that in [#7290](https://github.com/lightgbm-org/LightGBM/pull/7290) (in the Python package specifically, for [#6963](https://github.com/lightgbm-org/LightGBM/issues/6963)). Subscribe there to be notified how that goes.
> 
> It's looking like if it gets merged, it'd set up a good foundation for doing this generically across the CLI, R package, etc. (you did not tell us which language/interface you're using).

THINKS for reply!
